torch/csrc/jit/passes/utils/check_tensor_specializations.cpp

```diff
@@ -0,0 +1,67 @@
+#include <torch/csrc/jit/passes/utils/check_tensor_specializations.h>
+#include <torch/csrc/jit/jit_log.h>
+
+namespace torch {
```

Comment:
This is still leaking into source code instead of test

Reply:
As per our discussion, I changed the test to a C-based test so that we can have all of the testing code in test files.